Create ui-c8y-1023-36-1-add-a-fall-back-route-for-the-upgrade-module-10891-graft.md (#4231)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-36-1-add-a-fall-back-route-for-the-upgrade-module-10891-graft.md

---
date: ""

> title: Add a fallback route for not found routes

Reviewer: ```suggestion
title: Added fallback route for routing in hybrid applications
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64415
version: 1023.36.1
---

> The routing in the hybrid applications (applications that share angularjs and Angular code) could become inaccessible if certain routes where not found, potentially leaving users unable to access functionality. A fallback route has been added to the applications to ensure that users can always access this feature, even if the primary routing path is unavailable. This prevents users from encountering broken navigation paths when attempting to access features.

Reviewer: ```suggestion
Routing in hybrid applications (applications that share AngularJS and Angular code) could become inaccessible if certain routes are not found, potentially leaving users unable to access functionality. A fallback route has been added to the applications to ensure that users can always access this feature, even if the primary routing path is unavailable. This prevents users from encountering broken navigation paths when accessing features.
```
